apps/sms/locales/sms.en-US.properties

```diff
@@ -315,6 +315,10 @@ contact-photo.ariaLabel = Contact photo
 unread-thread.ariaLabel = Unread
 has-draft.ariaLabel = Has draft
 is-draft.ariaLabel = Draft
+optionsButton.ariaLabel = Options
+settingsButton.ariaLabel = Settings
+composeLink.ariaLabel = Compose
+callButton.ariaLabel = Call
```

Comment:
Should we have longer labels ?
Like:
* options => Display available actions
* settings => Open the settings panel
* compose => Compose a new message
* call => Call this contact/phone number (depending on whether we have a known contact) -- but maybe it's not easy and in that case "Call this contact" could work in both cases IMO